src/rendering/renderers/shared/system/AbstractRenderer.ts

```diff
@@ -301,6 +301,10 @@ export class AbstractRenderer<
             options.transform = options.container.localTransform;
         }
 
+        // lets ensure this object is a render group so we can render it!
+        // the renderer only likes to render - render groups.
+        options.container.enableRenderGroup();
```

Comment:
Would we need to disable the render group after the render?

Reply:
no, not for now - we are assuming that things will generally be rendered multiple times, setting it to false after a render will cause everything to rebuild each frame. 

But i do agree that this is kind of a hidden side effect. Lets think of a better way to handle this outside of this PR